prov/gni/test/rdm_sr.c

```diff
@@ -474,6 +497,33 @@ void rdm_sr_xfer_for_each_size(void (*xfer)(int len), int slen, int elen)
 	}
 }
 
+static inline void rdm_sr_check_err_cqe(struct fi_cq_err_entry *cqe, void *ctx,
+					uint64_t flags, void *addr, size_t len,
```

Comment:
Thanks for catching this Jim.

> Does the err_cqe have all the proper fields filled out for a source addr error?

This is checked within `rdm_sr_check_canceled` -- we _check_ to see if the `err` field is set to `FI_EADDRNOTAVAIL` if it is, then the application must do an AV insertion to progress the CQs. `rdm_sr_check_err_cqe` should not perform this check as it is simply checking for a valid error CQE entry.

> If so, how have you verified it?
I will update `rdm_sr_check_err_cqe` to check _all_ the relevant fields of the error CQE.